locations/fblipke.yml

```diff
@@ -0,0 +1,75 @@
+---
+location: fblipke
+location_nice: FBLipke - Freifunk Berlin
+latitude: 52.484948320
+longitude: 13.443380903
+altitude: 42
+height: 13
+contact_name: 'Guido L.'
+contact_nickname: 'LipkeGu'
+contacts:
+  - 'contact@fblipke.de'
+hosts:
+  - hostname: fblipke-core
+    role: corerouter
+    model: "x86_64"
+    wireless_profile: freifunk_default
```

Comment:
does your core have wireless interfaces? if not remove this line as it is only required for core routers that provide wireless interfaces.